src/Compilers/Core/Portable/Operations/CommonConversion.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents the common, language-agnostic elements of a conversion.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
```

Comment:
I'll just remove it. Can't really implement a struct :). I'll remark that we reserve the right to change it in the future.